lib/reporters/base.js

```diff
@@ -28,9 +28,7 @@ var isatty = tty.isatty(1) && tty.isatty(2);
  * Enable coloring by default, except in the browser interface.
  */
 
-exports.useColors =
-  !process.browser &&
-  (supportsColor.stdout || process.env.MOCHA_COLORS !== undefined);
+exports.useColors = !process.browser && supportsColor.stdout;
```

Comment:
How about this?

```js
exports.useColors = supportsColor && supportsColor.stdout;
```

Because `process.browser` is already tested in [L13](https://github.com/mochajs/mocha/pull/3665/files#diff-5c8a5788ecd4b4af2a29bc27ac4ae986R13).

Reply:
That's a good suggestion 👍 – thanks @outsideris

Reply:
In test cases value of ```useColors``` is considered false when value of ```process.browser``` is not null. And if we use ```exports.useColors = supportsColor && supportsColor.stdout; ``` then value of the ```useColors``` will be null when value of ```process.browser``` is not null. And that will be not correct behavior. So either we can  write ```exports.useColors = supportsColor != null && supportsColor.stdout;``` or leave it as it is.

Reply:
Leave it as is then -- more important _not_ to break anything...